### PR TITLE
Wrap stored accessors in parameters

### PR DESCRIPTION
**Pull request by sngeth**
### Summary

Attempt to improve wrapped parameters by wrapping Rails 5 Store Accessors referenced in Issue #26708

### CODE REVIEW

File: actionpack/test/controller/params_wrapper_test.rb [hunk 1]
```diff
@@ -34,7 +34,9 @@ def self.attribute_names
     end
   end
 
-  class Person
+  class Person < ActiveRecord::Base
```
Comment (maclover7) [inline actionpack/test/controller/params_wrapper_test.rb]: I don't believe `ActiveRecord` is available in Action Pack tests, you'll have to manually list out the methods you want 😬

**Comment by maclover7** [comment]
@kaspth thanks for the r?, but I don't have commit privileges :)

**Comment by kaspth** [comment]
@maclover7 nope, but try reviewing as thorough you can and then ping one of us who have commit :)

**Comment by rafaelfranca** [comment]
@sngeth Could you squash your commits?

**Comment by sngeth** [comment]
Squash and rebased from master @ 843345cae59b73e50af685ac1303a419a06b4c91 but it looks like there is an existing failure in ActionPack

**Comment by rafaelfranca** [comment]
Oops, it is my fault. I'll fix it.

**Comment by rafaelfranca** [comment]
Backported in 72c0667676

**Pull request merged**